railties/test/application/console_test.rb

```diff
@@ -110,6 +110,22 @@ def test_sandbox_by_default_with_development_environment
     @primary.puts "quit"
   end
 
+  def test_prompt_is_properly_set
+    options = "-e test"
+    spawn_console(options)
+
+    write_prompt "a = 1", "app-tempalte(test)> a = 1"
```

Comment:
I think this should be
```suggestion
    write_prompt "a = 1", "app-template(test)> a = 1"
```

which is causing the test to fail: https://buildkite.com/rails/rails/builds/106824#018f40aa-1a1e-4326-86fa-961aeaaf20dc/1286-1294

Reply:
Oops, thanks!